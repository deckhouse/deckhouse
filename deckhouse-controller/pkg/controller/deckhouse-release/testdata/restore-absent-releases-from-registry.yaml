apiVersion: deckhouse.io/v1alpha1
kind: DeckhouseRelease
metadata:
  name: v1.57.3
spec:
  version: "v1.57.3"
status:
  phase: Deployed
