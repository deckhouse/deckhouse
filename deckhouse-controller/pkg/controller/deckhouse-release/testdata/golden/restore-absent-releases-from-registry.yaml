---
apiVersion: deckhouse.io/v1alpha1
approved: false
kind: DeckhouseRelease
metadata:
  creationTimestamp: null
  name: v1.57.3
  resourceVersion: "999"
spec:
  version: v1.57.3
status:
  approved: false
  message: ""
  phase: Deployed
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
approved: false
kind: DeckhouseRelease
metadata:
  annotations:
    release.deckhouse.io/isUpdating: "false"
    release.deckhouse.io/notified: "false"
  creationTimestamp: null
  name: v1.58.1
  resourceVersion: "1"
spec:
  changelogLink: https://github.com/deckhouse/deckhouse/releases/tag/v1.58.1
  version: v1.58.1
status:
  approved: false
  message: ""
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
approved: false
kind: DeckhouseRelease
metadata:
  annotations:
    release.deckhouse.io/isUpdating: "false"
    release.deckhouse.io/notified: "false"
  creationTimestamp: null
  name: v1.59.3
  resourceVersion: "1"
spec:
  changelogLink: https://github.com/deckhouse/deckhouse/releases/tag/v1.59.3
  version: v1.59.3
status:
  approved: false
  message: ""
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
approved: false
kind: DeckhouseRelease
metadata:
  annotations:
    release.deckhouse.io/isUpdating: "false"
    release.deckhouse.io/notified: "false"
  creationTimestamp: null
  name: v1.60.2
  resourceVersion: "1"
spec:
  changelogLink: https://github.com/deckhouse/deckhouse/releases/tag/v1.60.2
  version: v1.60.2
status:
  approved: false
  message: ""
  transitionTime: null
